lib/shared/messages.js

```diff
@@ -52,6 +52,13 @@ module.exports = {
       'You are currently flashing a drive. Closing Etcher may leave',
       'your drive in an unusable state.\n\n',
       'Are you sure you want to close Etcher?'
+    ].join(' ')),
+
+    looksLikeWindowsImage: _.template([
+      'It looks like you are trying to write a Windows image.',
+      'Windows images require special treatment that is not currently built into Etcher in order to be made bootable.',
+      'It is recommended to use a specialised tool such as <a href="#" os-open-external="https://rufus.akeo.ie">Rufus</a>',
```

Comment:
Yeah, let's keep it inline for now. I might be wrong, but I though the `<a>` works just fine for these cases. What is its effect?